content/blog/2017/2017-01-17-Jenkins-is-upgrading-to-Java-8.adoc

```diff
@@ -0,0 +1,41 @@
+---
+layout: post
+title: "Jenkins Upgrades To Java 8"
+tags:
+- java8
+- upgrade
+author: batmat
+---
+
+**In the next few months, Jenkins will require Java 8 as its runtime.**
+
+Back in link:/blog/2016/11/03/2016-11-22-what-jvm-versions-are-running-jenkins-the-return/[last November, we discussed interesting statistics showing that Jenkins was now running Java 8 on a majority of its running instances].
```

Comment:
This paragraph is logically connected to https://github.com/jenkins-infra/jenkins.io/pull/545/files#diff-5585914dc691986da9b7537331fe0c1aR31 .  It's more reasons why we're upgrading. 

Can you move these two lines to just before line 31?

Or move this information into the list:

```
* Over 55% of all Jenkins instances are already running on Java 8.
* Over 70% of all Jenkins 2.x instances are running on Java 8.
```